features/account/customer_account/address_book/adding_address.feature

```diff
@@ -10,8 +10,8 @@ Feature: Adding a new address to the book
 
     @ui
     Scenario: Adding address to address book
-        Given I want to add a new address to my address book
+        Given I am adding a new address to my book
```

Comment:
Why you changed that? Right now in all scenarios we use `I want to ...`, if we decide to change this convention it should be corrected everywhere else.